python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -379,16 +382,21 @@ def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList, bgInfoL
         @param tempExpRefList: List of data references to tempExp
         @param imageScalerList: List of image scalers
         @param weightList: List of weights
-        @param bgInfoList: List of background data from background matching
+        @param bgInfoList: List of background data from background matching, or None
+        @param altMaskList: List of alternate masks to use than those on disk, or None
```

Comment:
It should be reworded as John indicates to mean other than those stored with the tempExp.